azure-pipelines-integration.yml

```diff
@@ -0,0 +1,25 @@
+# Branches that trigger a build on commit
+trigger:
+- master
+
+# Branches that trigger builds on PR
+pr:
+- master
+
+jobs:
+
+- job: Windows
+  pool: dotnet-external-temp
+  strategy:
+    maxParallel: 4
+  timeoutInMinutes: 120
+  steps:
+    - script: eng\integration-test.cmd -logPath '$(Build.SourcesDirectory)\artifacts\log\integration' -testPath '$(Build.SourcesDirectory)\temp'
```

Comment:
Why not define separate runs for each repository so they can happen in parallel?

```yaml
  strategy:
    maxParallel: 9
    matrix:
      format:
        _repo: "https://github.com/dotnet/format"
      roslyn:
        _repo: "https://github.com/dotnet/roslyn"
      cli:
        _repo: "https://github.com/dotnet/cli"
      project-system:
        _repo: "https://github.com/dotnet/project-system"
      msbuild:
        _repo: "https://github.com/Microsoft/msbuild"
      corefx:
        _repo: "https://github.com/dotnet/corefx"
      Blazor:
        _repo: "https://github.com/aspnet/Blazor"
      EntityFrameworkCore:
        _repo: "https://github.com/aspnet/EntityFrameworkCore"
      EntityFramework6:
        _repo: "https://github.com/aspnet/EntityFramework6"
```
you would just need to change the script to accept a repo URL
```yaml
- script: eng\integration-test.cmd -logPath '$(Build.SourcesDirectory)\artifacts\log\integration' -testPath '$(Build.SourcesDirectory)\temp' -repo $(_repo)
```